tutorials/nginx-ingress-gke/index.md

```diff
@@ -219,7 +231,15 @@ solution on Kubernetes Engine.
 
 ![image](https://storage.googleapis.com/gcp-community/tutorials/nginx-ingress-gke/Nginx%20Ingress%20on%20GCP%20-%20Fig%2002.png)
 
-From the Cloud Shell, deploy an NGINX controller Deployment and Service by running the following command:
+#### Deploy NGINX Ingress Controller with RBAC enabled
+If your Kubernetes cluster has RBAC enabled, From the Cloud Shell, deploy an NGINX controller Deployment and Service by running the following command:
+
+```
+helm install --name nginx-ingress stable/nginx-ingress --set rbac.create=true
+```
+
+#### Deploy NGINX Ingress Controller with RBAC disabled
+If your Kubernetes cluster has RBAC disabled, From the Cloud Shell, deploy an NGINX controller Deployment and Service by running the following command:
```

Comment:
make the "F" lowercase: '...disabled, from the Cloud Shell...'

Reply:
done